### DISCUSSION TITLE
Atomic Slot Recipes not working in Next.js

### DISCUSSION THREAD

**Discussion by kopestik** [chakra-ui/panda#2622] [Q&A] [answered]
I tried PandaCSS and just copy-pasted the Atomic Slot Recipe code from the docs into a Next.js project. However, it doesn't seem to be working:

Config:
https://github.com/kopestik/test-panda/blob/master/panda.config.ts

Code:
Recipe: https://github.com/kopestik/test-panda/blob/master/src/recipes/checkbox.ts
Usage: https://github.com/kopestik/test-panda/blob/master/src/app/page.tsx

Result:
![image](https://github.com/chakra-ui/panda/assets/35700005/a6e72fc2-df99-4950-94ca-11ca9795bfbf)

Am I missing something or doing something wrong?

**Comment by astahmer** [comment, marked as answer]
in layout.tsx you `import "./globals.css";` but that file only contains [the `@layers` order](https://panda-css.com/docs/installation/postcss#configure-the-entry-css-with-layers)
since you didnt use the postcss plugin, the CSS is never injected
you can do so like this https://panda-css.com/docs/installation/postcss#add-panda-to-your-postcss-config

or you can use the CLI only (run `panda` command) and replace the `import "./globals.css";` to `import "../../styled-system/styles.css";`
